### PR TITLE
feat(groundstation): Adds initial groundstation L2 constructs

### PR DESCRIPTION
**Pull request by kochie**
----

### All Submissions:

* [x] Have you followed the guidelines in our [Contributing guide?](https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md)

### Adding new Unconventional Dependencies:

* [ ] This PR adds new unconventional dependencies following the process described [here](https://github.com/aws/aws-cdk/blob/master/CONTRIBUTING.md/#adding-new-unconventional-dependencies)

### New Features

This PR adds L2 constructs for AWS Ground Station. The constructs include individual config, mission profile, and endpoint groups.

- `MissionProfile`
- `DataflowEndpointGroup`
- `AntennaDownlinkConfig`
- `AntennaDownlinkConfigDemodDecodeConfig`
- `AntennaUplinkConfig`
- `DataflowEndpointConfig`
- `S3RRecordingConfig`
- `TrackingConfig`
- `UplinkEchoConfig`

* [x] Have you added the new feature to an [integration test](https://github.com/aws/aws-cdk/blob/master/INTEGRATION_TESTS.md)?
	* [x] Did you use `cdk-integ` to deploy the infrastructure and generate the snapshot (i.e. `cdk-integ` without `--dry-run`)?
	* Caveat to the above. The config constructs for Uplink require permission to broadcast on a particular frequency which is something that I don't have. Therefore it won't be possible to run these integration tests for `UplinkEchoConfig` and `AntennaUplinkConfig` without those permissions.

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

File: packages/@aws-cdk/aws-groundstation/lib/mission_profile.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+export interface MissionProfileProps {
+  ContactPostPassDurationSeconds?: number;
+  ContactPrePassDurationSeconds?: number;
+  DataflowEdges: DataflowEdge[];
+  MinimumViableContactDurationSeconds: number;
+  Name: string;
+  TrackingConfig: TrackingConfig;
+}
+
+export abstract class MissionProfie extends Resource implements IMissionProfile {
```
Comment (planteen) [inline packages/@aws-cdk/aws-groundstation/lib/mission_profile.ts:22]: Typo here: `MissionProfie` should be `MissionProfile`

**Comment by comcalvi** [comment]
Thanks for opening this PR! This is a phenomenal contribution, and I can't wait to see it merged. Unfortunately we're still in the process of updating our contributing guide to reflect this, but we're asking contributors to submit RFCs for new L2s before we can review the PR itself. Please see https://github.com/aws/aws-cdk/pull/20113#issuecomment-1112401243 for some additional details. If you'd like to lead the RFC effort, please do open one!

**Comment by kochie** [comment]
Hey @comcalvi thanks for the response. Honestly having RFC's for each L2 construct sounds like a good idea and for something like groundstation I probably should of made one before doing this PR since there are a lot of different ways it can be built (and a lot of ideas that I think can be implemented). I'll start working on one and link here.

**Comment by TheRealAmazonKendra** [comment]
This PR has been deemed to be abandoned, and will be automatically closed. Please create a new PR for these changes if you think this decision has been made in error.

**Pull request closed without merging**